schemas/v3.0/schema.yaml

```diff
@@ -0,0 +1,1400 @@
+          - $ref: '#/definitions/Schema'
+          - $ref: '#/definitions/Reference'
+      example: {}
+    patternProperties:
+      '^x-': {}
+    additionalProperties: false
+
+  ParameterWithSchemaWithExampleInQuery:
+    type: object
+    required:
+      - name
+      - in
+      - schema
+    properties:
+      name:
+        type: string
+      in:
+        type: string
+        enum:
+          - query
+      description:
+        type: string
+      required:
+        type: boolean
+        default: false
+      deprecated:
+        type: boolean
+        default: false
+      allowEmptyValue:
+        type: boolean
+        default: false
+      style:
+        type: string
+        enum:
+          - form
+          - spaceDelimited
+          - pipeDelimited
+          - deepObject
+        default: form
+      explode:
+        type: boolean
+      allowReserved:
+        type: boolean
+        default: false
+      schema:
+        oneOf:
+          - $ref: '#/definitions/Schema'
+          - $ref: '#/definitions/Reference'
+      example: {}
+    patternProperties:
+      '^x-': {}
+    additionalProperties: false    
+
+  ParameterWithSchemaWithExampleInHeader:
+    type: object
+    required:
+      - name
+      - in
+      - schema
+    properties:
+      name:
+        type: string
+      in:
+        type: string
+        enum:
+          - header
+      description:
+        type: string
+      required:
+        type: boolean
+        default: false
+      deprecated:
+        type: boolean
+        default: false
+      allowEmptyValue:
+        type: boolean
+        default: false
+      style:
+        type: string
+        enum:
+          - simple
+        default: simple
+      explode:
+        type: boolean
+      allowReserved:
+        type: boolean
+        default: false
+      schema:
+        oneOf:
+          - $ref: '#/definitions/Schema'
+          - $ref: '#/definitions/Reference'
+      example: {}
+    patternProperties:
+      '^x-': {}
+    additionalProperties: false    
+
+  ParameterWithSchemaWithExampleInCookie:
+    type: object
+    required:
+      - name
+      - in
+      - schema
+    properties:
+      name:
+        type: string
+      in:
+        type: string
+        enum:
+          - cookie
+      description:
+        type: string
+      required:
+        type: boolean
+        default: false
+      deprecated:
+        type: boolean
+        default: false
+      allowEmptyValue:
+        type: boolean
+        default: false
+      style:
+        type: string
+        enum:
+          - form
+        default: form
+      explode:
+        type: boolean
+      allowReserved:
+        type: boolean
+        default: false
+      schema:
+        oneOf:
+          - $ref: '#/definitions/Schema'
+          - $ref: '#/definitions/Reference'
+      example: {}
+    patternProperties:
+      '^x-': {}
+    additionalProperties: false  
+
+  ParameterWithSchemaWithExamples:
```

Comment:
I seem to be having problems with these - as neither `example` or `examples` is required in `ParameterWithSchemaWithExampleInQuery` or `ParameterWithSchemaWithExamplesInQuery` and there is no `ParameterWithSchemaWithoutExampleInQuery` type, how is the correct type to match the `oneOf` to be determined?

I temporarily changed `examples` to be a required property in `ParameterWithSchemaWithExamplesIn*`.